cadc-util/src/main/java/ca/nrc/cadc/util/RsaSignatureVerifier.java

```diff
@@ -165,6 +163,15 @@ public RsaSignatureVerifier(File keyFile) {
         }
         init(keyFile);
     }
+
+    public RsaSignatureVerifier(byte[] keyContent) {
+        init(keyContent);
+    }
+
+    public RsaSignatureVerifier(byte[] keyContent, boolean isPrivateKey) {
```

Comment:
I don't really grok what these ctor with `boolean  isPrivateKey` arg are for... if that arg is true, there is no init.

Reply:
This happens because the default deprecated actor requires the key file to be presented. Unfortunately, this ctor is still used in a few places. I could clean it but it requires other package releases. It looks ugly but I couldn't find a better solution. In the future, we need to change checkstyle to fail on deprecated code.